src/LibraryViewExtension/Handlers/IconResourceProvider.cs

```diff
@@ -27,7 +27,8 @@ public override Stream GetResource(IRequest request, out string extension)
         {
             extension = "png";
             var url = new IconUrl(new Uri(request.Url));
-            var libraryCustomization = LibraryCustomizationServices.GetForAssembly(url.Path, pathManager, true);
+            var path = Path.GetFullPath(url.Path);
```

Comment:
Some comment might help, because using Path.GetFullPath on a url path is a bit unexpected.